components/html-block/html-block.js

```diff
@@ -188,7 +199,8 @@ class HtmlBlock extends LitElement {
 		super.updated();
 		if (this._contextObserverController && this._contextObjectHasChanged()) {
 			const template = this.querySelector('template');
```

Comment:
It is intriguing that above, it queries via `assignedNodes` to get the template, whereas here it just queries the lightDOM using `this.querySelector`.  Technically, the resolution that `assignedNodes({ flatten: true })` is more desirable.  I wonder why this line doesn't do that.

Reply:
Ah, mostly because when I originally implemented this, I thought `assignedNodes` was specific to the `slotchange` event and not just an API on the slot itself. I can change this.